studio/components/interfaces/Organization/InvoicesSettings/InvoicesSettings.tsx

```diff
@@ -124,8 +128,23 @@ const InvoicesSettings = () => {
                       <Table.td>
                         <p>{x.number}</p>
                       </Table.td>
+                      <Table.td>
+                        <InvoiceStatusBadge status={x.status} />
+                      </Table.td>
                       <Table.td className="align-right">
                         <div className="flex items-center justify-end space-x-2">
+                          {['uncollectible', 'open'].includes(x.status) && (
+                            <Button
```

Comment:
Wrap this within a

import Link from 'next/link'

```
<Link href="xxx">
  <a target="_blank">
    <Button />
  <a/>
</Link>
```

then remove the href on the button